src/test/java/core/basesyntax/service/impl/ReportMakerServiceImplTest.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.service.ReportMakerService;
+import org.junit.After;
+import org.junit.Test;
+
+public class ReportMakerServiceImplTest {
+    private static final String HEADER = "fruit,quantity";
+    private static final String FRUIT_AMOUNT_LINE = "apple,50";
+    private static final String FRUIT = "apple";
+    private static final int QUANTITY = 50;
+    private static final String VALID_REPORT = HEADER + System.lineSeparator() + FRUIT_AMOUNT_LINE;
+    private static final ReportMakerService reportMaker = new ReportMakerServiceImpl();
```

Comment:
Try to follow one code style, if you create all services in `beforeClass` method - do it everywhere